src/arch/armv8/aarch32/boot.S

```diff
@@ -154,6 +154,19 @@ _set_master_cpu:
     /* If this is the cpu master, clear bss */
     cmp r9, #0
     bne 1f
+
+#ifdef MEM_NON_UNIFIED
+    /* Copy data from RX to RWX */
+    ldr r7, =_data_lma_start        // LMA start
+    ldr r11, =_data_vma_start       // VMA start
+    ldr r12, =_image_load_end       // LMA end
+    bl copy_data
+
+    ldr r1, =_data_vma_start
+    ldr r3, =data_addr
+    str r1, [r3] // store image load address in img_addr
+#endif
```

Comment:
Why is this added in this PR? Shoudn't the copy of data be unaltered in this commit?